src/Algebra/Properties/CommutativeMagma/Divisibility.agda

```diff
@@ -28,7 +28,7 @@ x∣xy : ∀ x y → x ∣ x ∙ y
 x∣xy x y = y , comm y x
 
 xy≈z⇒x∣z : ∀ x y {z} → x ∙ y ≈ z → x ∣ z
-xy≈z⇒x∣z x y xy≈z = ∣-respʳ xy≈z (x∣xy x y)
+xy≈z⇒x∣z x y xy≈z = ∣-respʳ-≈ xy≈z (x∣xy x y)
```

Comment:
I don't understand why this doesn't have the same redundancy problem as the one in `Magma`?

Reply:
Good question! (Probably an additional appeal to `comm`?)

Reply:
Indeed, the normal form of the RHS here is simply `y , trans (comm y x) xy≈z`, so there isn't the same opportunity for `trans refl` to 'reduce'.

Reply:
In respect of the originating issue #2629 , though, ... examples such as these potentially give rise to subsequent 'redexes' of the form `trans (comm y x) (trans (comm x y) ...)` which should also be 'reduced' away.

Reply:
Agreed: but is there anything left to do here? I don't think so. So I'll mark this one as resolved.